src/entries/popup/components/CommandK/useCommands.tsx

```diff
@@ -742,6 +802,15 @@ export const useCommands = (
     [navigate, setSelectedToken],
   );
 
+  const selectSearchTokenAndNavigate = React.useCallback(
+    (asset: SearchAsset, to: To) => {
+      setSelectedToken();
+      saveSwapTokenToBuy({ token: asset as ParsedSearchAsset });
```

Comment:
This is ok for now since we're going to parse them later in swap and bridge page. We're doing this in other places like here https://github.com/rainbow-me/browser-extension/blob/b857ab6b003428ae96fb3b510efcfd6fb50186ab/src/entries/popup/pages/swap/SwapTokenInput/TokenDropdown/TokenToBuySection.tsx#L203